zeroconf/const.py

```diff
@@ -46,8 +45,10 @@
 _MDNS_ADDR = '224.0.0.251'
 _MDNS_ADDR_BYTES = socket.inet_aton(_MDNS_ADDR)
 _MDNS_ADDR6 = 'ff02::fb'
-with contextlib.suppress(OSError):  # can't use AF_INET6, IPv6 is disabled
+try:
     _MDNS_ADDR6_BYTES = socket.inet_pton(socket.AF_INET6, _MDNS_ADDR6)
+except OSError:  # can't use AF_INET6, IPv6 is disabled
+    _MDNS_ADDR6_BYTES = b'\xff\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xfb'
```

Comment:
```suggestion
    _MDNS_ADDR6_BYTES = None
```

I think `None` makes sense here if we know its not going to work.